package.json

```diff
@@ -3,34 +3,53 @@
   "name": "parent",
   "version": "0.0.0",
   "devDependencies": {
-    "colors": "^1.1.2",
+    "@types/chai": "^4.0.1",
+    "@types/chai-as-promised": "0.0.31",
+    "@types/mocha": "^2.2.41",
+    "@types/temp": "^0.8.29",
+    "@types/webdriverio": "^4.7.0",
+    "chai": "^4.1.0",
+    "chai-as-promised": "^7.1.1",
+    "concurrently": "^3.5.0",
+    "electron-mocha": "^3.5.0",
     "lerna": "^2.1.2",
-    "yo": "^2.0.0"
+    "mocha": "^3.4.2",
+    "nyc": "^11.0.3",
+    "rimraf": "^2.6.1",
+    "selenium-standalone": "^6.2.0",
+    "temp": "^0.8.3",
+    "ts-node": "^3.2.0",
+    "tslint": "^5.5.0",
+    "typedoc": "^0.8",
+    "typescript": "^2.4.1",
+    "uuid": "^3.1.0",
+    "wdio-mocha-framework": "^0.5.9",
+    "wdio-phantomjs-service": "^0.2.2",
+    "wdio-selenium-standalone-service": "0.0.8",
+    "wdio-spec-reporter": "^0.1.0",
+    "webdriverio": "^4.6.2"
   },
   "scripts": {
-    "prepare": "yarn run prepare:generator && yarn run prepare:packages",
-    "prepare:generator": "yarn run bootstrap:generator && yarn run build:generator && yarn run link:generator",
-    "bootstrap:generator": "lerna bootstrap --scope generator-theia -- --no-optional",
-    "build:generator": "lerna run prepare --scope generator-theia --stream",
-    "link:generator": "lerna exec --scope generator-theia --stream -- yarn link && yarn link generator-theia",
-    "prepare:packages": "yarn run bootstrap && yarn run clean && yarn run clean:rebuild && yarn run build",
-    "bootstrap": "node scripts/generate && node scripts/lerna bootstrap --scope \"@theia/*\" --hoist -- --no-optional",
-    "refresh": "node scripts/lerna clean --scope \"@theia/*\" --yes && yarn run bootstrap",
-    "clean": "node scripts/lerna run clean --scope \"@theia/*\" --parallel",
-    "build": "node scripts/lerna run build --scope \"@theia/*\" --stream",
-    "docs": "node scripts/lerna run docs --scope \"@theia/*\" --stream",
+    "prepare": "yarn run clean && yarn run rebuild:clean && yarn run build",
+    "clean": "node scripts/lerna run clean --parallel",
+    "build": "node scripts/lerna run build --stream",
+    "docs": "node scripts/lerna run docs --scope \"@theia/!(example-)*\" --stream",
     "test": "yarn run test:theia && yarn run test:electron && yarn run test:browser",
-    "test:theia": "node scripts/lerna run test --scope \"@theia/!(example-)*\" --parallel && yarn test:theia:slow",
-    "test:theia:slow": "node scripts/lerna run test:slow --scope \"@theia/terminal\" --parallel",
+    "test:theia": "node scripts/lerna run test --scope \"@theia/!(example-)*\" --parallel",
```

Comment:
why did you remove test:theia:slow?

Reply:
do you run it?

Reply:
I kept them for projects, e.g. `npx lerna run test:slow --scope @theia/terminal`

Reply:
Yes it was run by CI by default, I would like to keep it that way

Reply:
it is still run by CI, each package has `test` script that runs `test:fast` + `test:slow`

Reply:
isn't that failing because many pacakge do not have slow tests ?

Reply:
I would like to simplify package.json when it is possible and remove scripts which are just there without anybody calling them regularly

Reply:
it is never failed if there are no tests, dummy files because of test coverage

Reply:
Yes the dummy files work for the test:fast pattern but not for the :slow

Reply:
it was the same before this PR should be tackled differently.

Reply:
No before this PR I explicitly selection that test:slow should be run on package/terminal only , otherwise the test run would fail

Reply:
See: the scope here:  `test:theia:slow": "node scripts/lerna run test:slow --scope \"@theia/terminal\" --parallel",`